### ISSUE TITLE
[A11Y] The label of the datagrid "Available actions" button is not unique and descriptive

### ISSUE THREAD

**Issue by KingMario** [vmware/clarity#6567]
## Describe the bug

The label of the datagrid 'Available actions' button is not unique and descriptive.

## How to reproduce

1.  Go to https://stackblitz.com/edit/clarity-light-theme-v5-datagrid-simple?file=src%2Fapp%2Fapp.component.html
2.  Turn on screen reader such as NVDA for Windows or VoiceOver for Mac
3.  Use Tab key to focus on the vertical '...' button of each row of the datagrad
4.  Check if the announcement of "Available actions" button is unique or not

![image](https://user-images.githubusercontent.com/3956876/148877756-f05a0ba1-da14-45df-92b4-d96bfda8cd8f.png)

## Versions

**Clarity project:**

- [ ] Clarity Core
- [x] Clarity Angular/UI

**Clarity version:**

- [ ] v3.x
- [ ] v4.x
- [x] v5.x

**Framework:**

- [x] Angular
- [ ] React
- [ ] Vue
- [ ] Other:

**Comment by gnomeontherun** [comment]
@KingMario are these issues tracked by our internal a11y team? If so that is where we review and triage issues so we don't have duplicate issues. If they are not tracked by our a11y team, they should be. Adding @scroniser for visibility.
